src/App.js

```diff
@@ -1,8 +1,29 @@
 import React from "react";
 import "./index.css";
+import Contacts from "./Components/Contacts";
+import UserProfile from "./Components/UserProfile";
+import NavBar from "./Components/NavBar";
+import Posts from "./Components/Posts";
+import Post from "./Components/Post";
+import ContactUserCard from "./Components/ContactUserCard";
 
 const App = () => {
-  return <p>Hello, world!</p>;
+  return (
+    <div className="App">
+      <NavBar />
+      <UserProfile />
+      <Posts>
```

Comment:
Hey Theodore, nice work! However, just a quick note: Components (written in JSX tags), for now, do not have opening and closing tags, like HTML. They also are not nested within each other all in the App component. For example, all of your Post components should be imported and used within your Posts component. Posts is then imported into App. I think you may realize that now after all the work and review we did last week.